ggu/main.c

```diff
@@ -68,25 +76,21 @@ process_single_policy(const struct ggu_policy *policy, const struct ggu_config *
 	entry->u.ggu.state = policy->state;
 	rte_memcpy(&entry->u.ggu.flow, &policy->flow, sizeof(entry->u.ggu.flow));
 
-	switch(policy->state) {
+	switch (policy->state) {
 	case GK_GRANTED:
 		entry->u.ggu.params.u.granted.tx_rate_kb_sec =
```

Comment:
Can't we replace these lines for a `rte_memcpy()` now?

If so, the same goes for `case GK_DECLINED:`.